packages/components/src/resizable-box/README.md

```diff
@@ -41,8 +41,8 @@ const Edit = ( props ) => {
 			} }
 			onResizeStop={ ( event, direction, elt, delta ) => {
 				setAttributes( {
-					height: parseInt( height + delta.height, 10 ),
-					width: parseInt( width + delta.width, 10 ),
+					height: parseInt( elt.style.height + delta.height, 10 ),
```

Comment:
Hi @scruffian, the height and width variables are the current height and width attributes I guess the previous example would work. Any specific reason to forever reading the values from elt.style?